### DISCUSSION TITLE
windows store has more then one

### DISCUSSION THREAD

**Discussion by dancasmassonos** [JanDeDobbeleer/oh-my-posh#2402] [General]
windows store has more then one

![image](https://user-images.githubusercontent.com/91165638/173091785-c80681c3-6203-4a9a-9aaf-56e7540a5b5b.png)

Which is the correct one to install?

**Comment by JanDeDobbeleer** [comment]
They're the same 😁 But winget is more up-to-date.

**Reply by dancasmassonos** [reply]
so i change the command to 
winget install 'oh my posh' ?
When I tired that before PS and everything else didn't see the install

**Reply by JanDeDobbeleer** [reply]
It seems that with the launch of the store app we need to revert back to:

```
winget install JanDeDobbeleer.OhMyPosh
```

**Reply by bhagerty** [reply]
@JanDeDobbeleer - I made a pull request on the docs page to make this change.

**Reply by JanDeDobbeleer** [reply]
@bhagerty noticed it, the commit message will bounce but I'll fix that.

**Comment by dancasmassonos** [comment]
Also, thanks for your hard work again :)

**Comment by dancasmassonos** [comment]
I got it to work.
